Automattic/wp-calypso
# Add a new banner for blaze on woo express implementation

Related to 
a8c-dsp issue 789

We have a variant of the header for Woo:


![Screenshot 2024-01-03 at 15 52 55](https://github.com/Automattic/wp-calypso/assets/1416426/4975c83c-d4f3-412d-b165-30c301f42909)



## Proposed Changes

Added a new react component with the changes. 
added a few ternary conditions in the existing code to render different text in relation to the entrypoint of the blaze dashboard
added a screenshot above as reference for translation purposes

changes are not going to be in effect yet (so if you preview the auto generated PR preview nothing should be changed)

## Testing Instructions

in order to actually test this branch locally you need to add the below line of code in your `config/development.json` file

`"is_running_in_woo_site": true,`

to test the banner you also need to hardcode the value of `showBanner`
line 201 of `client/my-sites/promote-post-i2/main.tsx` and set 
`const showBanner = true;`

please test the banner in different screen sizes and make sure that it works as expected



*

## Pre-merge Checklist
